packages/xod-project/src/patch.js

```diff
@@ -738,6 +738,24 @@ export const upsertComments = def(
 //
 // =============================================================================
 
+const listLinksNotFromDeferredNodes = def(
+  'listLinksNotFromDeferredNodes :: Patch -> [Link]',
+  R.converge(
+    (deferredNodeIds, links) => R.reject(
+      R.pipe(Link.getLinkOutputNodeId, isAmong(deferredNodeIds)),
+      links
+    ),
+    [
+      R.compose(
+        R.map(Node.getNodeId),
```

Comment:
Looks like `getDeferredNodeIds` function, that could be reused in the transpiler.js `getDeferredNodeIds` :)